src/datanode/src/instance/sql.rs

```diff
@@ -139,15 +138,16 @@ impl Instance {
                 unimplemented!("SHOW CREATE TABLE is unimplemented yet");
             }
             QueryStatement::Sql(Statement::Use(db)) => {
+                let (catalog, schema) =
+                    servers::parse_catalog_and_schema_from_client_database_name(&db);
```

Comment:
For use statement, our `use <catalog>-<schema>` is not a valid sql because of `-`. I suggest to support `use <schema>` only and do not allow user to switch catalog after logged in.